src/configure/configure.ts

```diff
@@ -695,8 +695,8 @@ class Orchestrator {
             await provisioningConfigurer.postSteps(provisioningConfiguration, (draftProvisioningPipeline.result.pipelineConfiguration as DraftPipelineConfiguration), this.inputs);
 
             // All done, now browse the pipeline
-            telemetryHelper.setCurrentStep('BrowsingPipeline');
-            await provisioningConfigurer.browseQueuedPipeline();
+            telemetryHelper.setCurrentStep('BrowseQueuedWorkflow');
```

Comment:
This telemetry might already be in use in existing queries/dashboards. Please do not change unles absolute necessary.

Reply:
Done